sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<p>Remove the if else block. With the ability for the user to dynamically update the profiling config, it may be difficult to track which steps have been profiled at a glance. Two options:</p><ul><li>Let the user pick the step which step they want to see profiling stats for (they have to know which steps they profiled).</li><li>Make an assumption based on the initial python profiling config and assume that those steps were still profiled. In that case, we can just pick the start step as the step to view stats for here.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>fixed the else block to pick up the start step of Python profiling, so it automatically picks up where the Python profiling started from</p>